_layouts/location.html

```diff
@@ -1,41 +1,305 @@
----
-layout: default 
 --- 
-<!-- create a var called promos, split it into an array -->
-{% assign promos = "" | split: "" %}
-<!-- loop on the page's happenings (locations) -->
-{% for happening in site.promos %}
-  <!-- search through the promos collection for the specific happening and assign to promo -->
-  {% if happening.target_audience contains page.title %}
-  <!-- add back to new array called promos -->
-    {% assign promos = promos | push: happening %}
-  {% endif %}
-{% endfor %}
+layout: default 
+---
+<!-- Pull only the promos that exist for the specific location if any -->
+{% assign location_happenings = site.promos | where: "target_audience", page.title %}
 
 {% include _shared-header.html %}
 
-<div class="container">
-  <div class="card-deck cards-4x">
-    <div class="row">
-    <!-- Happenings -->
-    {% assign date_sorted_promos = promos | sort: "published_at" | reverse %}
-    {% for happening in date_sorted_promos %}
+<!-- Jumbotron section -->
+<div class="jumbotron jumbotron-xl flush-bottom" style="background-image: url('{{ page.bg_image }}');">
+  <div class="push-ends mobile-soft-sides">
+    <span class="font-family-base text-uppercase">
+      <strong>welcome to</strong>
+    </span>
+    <h1 class="title hidden-xs" style="font-size: 5.0rem;">crossroads
+      <br>{{ page.name | downcase }}</h1>
+    <h2 class="title font-size-h1 text-white visible-xs">crossroads
+      <br>{{ page.name | downcase }}</h2>
+  </div>
+</div>
+
+<!-- General Info Section -->
+<section class="bg-blue-dark soft-ends">
+  <div class="container push-ends">
+    <div class="row push-ends soft-ends mobile-flush-top mobile-hard-top">
+      <div class="col-sm-6 soft-sides">
+        <div class="lead">{{ page.content }}</div>
+        <div class="row push-ends soft-half-top mobile-hard-bottom">
+          <div class="col-sm-6">
+            <p>
+              <strong>Service Times</strong>
+              </br>
+              {{ page.service_times | markdownify | remove: '<p>'| remove: '</p>' }}
+            </p>
+          </div>
+          <div class="col-sm-6">
+            <p>
+              <strong>Address</strong>
+              </br>
+              {{ page.address | markdownify | remove: '<p>' | remove: '</p>' }}
+              </br>
+              <a href="{{ page.map_url }}"
+                target="_blank">Get directions</a>
+            </p>
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-sm-12">
+            {% for series in site.series limit: 1 %}
```

Comment:
I think we want `{% assign current_series = site.series | sort: "date" | reverse | first %}` here and then use `current_series` instead of `series`